### ISSUE TITLE
TracePlugin not working for bottle + python 2.7.x

### ISSUE THREAD

**Issue by rodrigocollavo** [DataDog/dd-trace-py#438]
Hi, we have a backend using python 2.7.x, Im integrating Datadog APM following the documentation and I have the following error:
```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/bottle.py", line 764, in _handle
    return route.call(**args)
  File "/usr/local/lib/python2.7/site-packages/bottle.py", line 1575, in wrapper
    rv = callback(*a, **ka)
  File "/usr/local/lib/python2.7/site-packages/ddtrace/contrib/bottle/trace.py", line 32, in wrapped
    resource = "%s %s" % (request.method, request.route.rule)
  File "/usr/local/lib/python2.7/site-packages/bottle.py", line 1237, in __getattr__
    raise AttributeError('Attribute %r not defined.' % name)
```

when I go to trace.py line 32, I find this:
`resource = "%s %s" % (request.method, request.route.rule)`

somehow request.route doesn't exist, I think that probably is related to a different python or bottle version being used on our end.
Anyway `route` is already provided as a parameter for this method (`TracePlugin.apply(self, callback, route):`) so there's no need to call `request.route.rule`, you can call `route.rule` instead,
`resource = "%s %s" % (request.method, route.rule)`

Otherwise let me know if you have more details about this error and what's the right way to solve it,
thanks

**Comment by palazzem** [comment]
Hello @rodrigocollavo ! thanks for reporting your problem. Your description of the problem is perfect, can you provide us your Bottle version? I want to reproduce the problem so that we have a regression test for that. Thank you very much!

**Comment by rodrigocollavo** [comment]
@palazzem we are using bottle 0.11.6

**Comment by palazzem** [comment]
@rodrigocollavo thank you for sharing! I made a PR to address the problem, it's under review here: https://github.com/DataDog/dd-trace-py/pull/439 . Can you confirm it solves your issue?

**Comment by rodrigocollavo** [comment]
I will give a a try soon, anyway I don't the change specified in the title of that PR #439

**Comment by rodrigocollavo** [comment]
confirmed, it's working correctly with the fix provided in #439

**Comment by palazzem** [comment]
Great! going to merge in `master` so that it can be released with the next bugfix! Thanks for your feedback!
